session/sessiondata_test.go

```diff
@@ -0,0 +1,259 @@
+package session
+
+import (
+	"testing"
+	"time"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+// TestGetAccessToken verifies that the session data access token is returned
+// correctly
+func TestGetAccessToken(t *testing.T) {
+
+	Convey("Given I have session data with an access token", t, func() {
+
+		accessToken := "Foo"
+
+		var sessionData SessionData
+
+		session := map[string]interface{}{
+			"signin_info": map[string]interface{}{
+				"access_token": map[string]interface{}{
+					"access_token": accessToken,
+				},
+			},
+		}
+		sessionData = session
```

Comment:
Is the session variable needed here? sessionData could be initialised as the map?